content/docs/hosting/BYOC.md

```diff
@@ -98,169 +99,178 @@ With Appbase.io Dashboard you can easily share cluster with your team members. E
 
 ---
 
-### Using AMI
-
-Arc is also available via an Amazon Machine Image (AMI) on the AWS Marketplace. Arc works running a `systemd` service. With the AMI, you can seamlessly deploy Arc on an AWS EC2 instance.
+### How to create Arc instance?
 
-Before Creating an AWS EC2 Machine using AMI, let's create an [Arc instance](https://arc-dashboard.appbase.io/install) which will give us access to [Arc Dashboard](https://arc-dashboard.appbase.io).
+Creating an Arc instance will enable you to access [Arc Dashboard](https://arc-dashboard.appbase.io). While following instance creation process, you will get an `ARC_ID`. This will be helpful while configuring Arc, with any of the deployment options listed below.
```

Comment:
The first sentence is out of context.

Reply:
In fact, getting ARC_ID is something a user needs to do prior to having their Arc Instance created. It shouldn't structurally be here as it's not part of the flow as such. A user only cares about this section if they get stuck, it doesn't add any conceptual meaning. We can put this towards the end of the doc.